docs/tutorials/Containers.md

```diff
@@ -0,0 +1,475 @@
+## Containers
+
+Containers are self-contained software archives, that hold everything from the tool you want to run, to libraries and runtimes it requires, and the operating system it runs on.
+ 
+Best Practise WDL and CWL define containers for their tasks to run in, to ensure reproducibility and portability - that running the same task on a different system will run the exact same software. 
+
```

Comment:
> Best practice

I'm not sure I agree that you can state about "best practice" here - there are inordinate numbers of workflow managers, and sometimes a workflow manager, period, is overly complex for the need (using containers). I would remove the bias and just say:

> To ensure reproducibility and portability, WDL and CWL can define containers for their tasks to run in. Using containers means that you can be confident about running the same software across different systems.

Reply:
That's an interesting point.

IMO I'd still declare it to be "best practice" for both WDL & CWL, but that's because IMO having one's workflow be portable is **also** "best practice". One might choose not to care for all sorts of reasons, but I think as a community we should strive to encourage that direction.

Regardless, I agree on `s/practise/practice` as we try to aim for 🇺🇸 spellings (much to the chagrin of @cjllanwarne )

Reply:
I reordered the sentence to mix both of your ideas.
> To promote reproducibility and portability, it's considered best practice to define containers for a WDL and CWL task to run in - this ensures that running the same task on a different system will run the exact same software.